Representable/representable
# FAQ & Glossary pages

**changes**
- added FAQ and glossary pages to site
- added new dropdown to navbar : "Learn More"
- changed navbar carets to font awesome style
- created models for FAQs and Glossary definitions
- enabled creation of FAQ and Glossary entries in django admin

**to test**
- python manage.py migrate
- python manage.py runserver
- go to django admin (http://127.0.0.1:8000/admin/)
- create multiple new objects for Frequently Asked Questions and Glossary Definitions
- check formatting and behavior on http://127.0.0.1:8000/faq/
- check formatting and behavior on http://127.0.0.1:8000/glossary/

**screenshots**
![image](https://user-images.githubusercontent.com/41943646/103309975-9ce61080-49ba-11eb-8165-1ba900a62790.png)
![image](https://user-images.githubusercontent.com/41943646/103309996-a7a0a580-49ba-11eb-8bd1-6083486beaef.png)
